template/README.md

```diff
@@ -1,137 +1,163 @@
-# Starter template for `fredrikhgrelland/hashistack`
+<!-- markdownlint-disable MD041 -->
+<p align="center">
+ <img width="100px" src="https://www.svgrepo.com/show/58111/cube.svg" align="center" alt="Vagrant-hashistack" />
+ <h2 align="center">Vagrant-hashistack Template</h2>
+ <p align="center">Starter template for <a href="https://github.com/fredrikhgrelland/vagrant-hashistack">fredrikhgrelland/vagrant-hashistack</a></p>
+<p align="center">
+    <a href="https://github.com/fredrikhgrelland/vagrant-hashistack-template/actions">
+      <img alt="Build" src="https://github.com/fredrikhgrelland/vagrant-hashistack-template/workflows/CI/CD/badge.svg" />
+    </a>
+    <a href="https://github.com/fredrikhgrelland/vagrant-hashistack/releases">
+      <img alt="Releases" src="https://img.shields.io/github/v/release/fredrikhgrelland/vagrant-hashistack-template?label=latest%20version" />
```

Comment:
Should we do this temporary fix so as we fetch the release version by pointing to ```vagrant-hashistack```, that would look better than the text ```no release or repo not found```?
And of course we can change this when we would have solved #270?
```suggestion
      <img alt="Releases" src="https://img.shields.io/github/v/release/fredrikhgrelland/vagrant-hashistack?label=latest%20version" />
```

Reply:
Yeah, sounds good, let's do that